src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorage.php

```diff
@@ -53,13 +48,16 @@ public function __construct($uploadPath, $segments, $logger = null)
     /**
      * {@inheritdoc}
      */
-    public function save($tempPath, $fileName, $version, $storageOption = null)
+    public function save($tempPath, $fileName, $preferredStorageOptions = null)
```

Comment:
if you upload a new fileVersion the old storageOptions of the file are sent with it.
e.g. on the localStorage a new version of the file will be in the same folder as the older version of the file.